OpsCenter_6.8_Release_Notes.md

```diff
@@ -1,5 +1,25 @@
 # Release notes for OpsCenter
 
+# Release Notes for OpsCenter 6.8.33
+
+## Backup Service
+* Encryption key collision during restore is now prevented and logged. Keys from system tables will no longer be included when restoring. (OPSC-17115)
+
+## Core
+* Added gossip configuration files to diagnostics tarball. (OPSC-17480)
+* Added {{alert_on_no_host_available}} to the {{[cassandra]}} section of the cluster conf. When set to true an alert will be triggered if opscenterd is unable to make a cql connection to the monitored cluster. (OPSC-17398)
+
+## CVE
+* Added {{redirect_host}} config value to the {{[webserver]}} section of {{opscenterd.conf}} to mitigate host header injection vulnerability. (OPSC-17265)
+
+## Monitoring
+* Fixed issue in diagnostic tarball generation when {{diagnostic_tarball_skip_instance_check}} parameter is True. (OPSC-17471)
+* Added the {{Swap}} value to the os-metrics memory.json file to identify if {{Swap}} is enabled from a diagnostics generated using OpsCenter. (OPSC-17405)
+* Resolved an issue that made it appear as if alerts were triggered on the configured threshold instead of over it. (OPSC-17446)
```

Comment:
Same comment as before. This fix was a super specific targeting `os-disk-usage`. Why not let the customer know about it?

Reply:
Although the jira is specific to `os-disk-usage`. but fix we made is generic for all metrics.

Reply:
What about:
> Fixed metrics computation causing alerts to be triggered when values were close but below their threshold.